### PR TITLE
TFAC-212 Add Click Overlay to MoneyRaised Counter

### PR DESCRIPTION
**Pull request by jedtan**

### CODE REVIEW

File: src/components/DashboardPopover.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <Popover
+      open={open}
+      anchorEl={anchorEl}
+      onClose={onClose}
+      anchorOrigin={{ horizontal: 'center', vertical: 'bottom' }}
+      transformOrigin={{ horizontal: 'center', vertical: 'top' }}
+      className={className}
+    >
+      <div
+        style={{
```
Comment (kmjennison) [inline src/components/DashboardPopover.js:17]: Move to class with `makeStyles`.

File: src/components/MoneyRaised.js [hunk 2]
```diff
@@ -31,9 +38,44 @@ const MoneyRaised = (props) => {
...
+        open={isPopoverOpen}
+        anchorEl={spanRef.current}
+        onClose={() => {
+          setIsPopoverOpen(false)
+        }}
+        className={classes.popover}
+      >
+        <div
+          style={{
+            padding: 12,
```
Comment (kmjennison) [inline src/components/MoneyRaised.js:57]: Use class styling.

File: src/components/__tests__/DashboardPopover.test.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  open: false,
+  onClose: () => {},
+})
+
+describe('DashboardPopover', () => {
+  it('renders without error', () => {
+    const DashboardPopover = require('src/components/DashboardPopover').default
+    const mockProps = getMockProps()
+    expect(() => {
+      shallow(<DashboardPopover {...mockProps} />).dive()
```
Comment (kmjennison) [inline src/components/__tests__/DashboardPopover.test.js:16]: Remove unneeded `.dive()` from tests.

**Review by kmjennison — CHANGES_REQUESTED** [review]

File: src/components/__tests__/MoneyRaised.test.js [hunk 4]
```diff
@@ -108,8 +112,33 @@ describe('MoneyRaised component', () => {
...
+    const defaultMockProps = getMockProps()
+
+    const wrapper = mount(<MoneyRaised {...defaultMockProps} />)
+    expect(wrapper.find(DashboardPopover).prop('open')).toBe(false)
+
+    wrapper.find(Button).first().simulate('click')
+    expect(wrapper.find(DashboardPopover).prop('open')).toBe(true)
+  })
+
+  it('popover onClose sets isPopoverOpen to false', () => {
```
Comment (jedtan) [inline src/components/__tests__/MoneyRaised.test.js:131]: This one is really stumping me as it's failing. Would you mind taking a look @kmjennison, hope I'm not missing something obvious.

File: src/components/MoneyRaised.js [hunk 5]
```diff
@@ -1,12 +1,20 @@
...
 import { currencyFormatUSD } from 'src/utils/formatting'
 import { makeStyles } from '@material-ui/core/styles'
 import { get } from 'lodash/object'
+import DashboardPopover from 'src/components/DashboardPopover'
+import Typography from '@material-ui/core/Typography'
+import Button from '@material-ui/core/Button'
 
 const useStyles = makeStyles((theme) => ({
   currencyText: { color: get(theme, 'palette.backgroundContrastText.main') },
+  popover: { marginTop: 12 },
```
Comment (kmjennison) [inline src/components/MoneyRaised.js:13]: Please reduce to 9px (aligns with impact notification layout).

File: src/components/MoneyRaised.js [hunk 6]
```diff
@@ -31,9 +42,39 @@ const MoneyRaised = (props) => {
   }, msPerPenny)
 
   return (
-    <span className={classes.currencyText}>
-      {currencyFormatUSD(currentMoneyRaised)}
-    </span>
+    <div>
+      <Button ref={spanRef} onClick={() => setIsPopoverOpen(true)}>
```
Comment (kmjennison) [inline src/components/MoneyRaised.js:46]: The button adds some additional margin that increased the menu spacing. Let's compensate for it so the menu stays approx the same: I think you can remove the `moneyRaisedContainer` class margin in index.js.

**Review by kmjennison — CHANGES_REQUESTED** [review]
Looks good! Just want to make some tiny style tweaks.

**Pull request merged**